src/components/Book.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Button, Container } from 'react-bootstrap';
+import InputBook from './InputBook';
+
+const Book = () => (
```

Comment:
- [ ] The `Book` component should take the book data as a **prop** and renders the title and the author, with a button to remove the book, kindly consider implementing this so your component can take as many books as needed not just hard-coded books